migration/migrator/data/submitty_db.sql

```diff
@@ -211,6 +211,7 @@ ALTER TABLE ONLY sessions
 ALTER TABLE ONLY courses_registration_sections
     ADD CONSTRAINT courses_registration_sections_fkey FOREIGN KEY (semester, course) REFERENCES courses(semester, course) ON UPDATE CASCADE;
 
+ALTER TABLE ONLY users ALTER COLUMN user_email DROP NOT NULL;
```

Comment:
No need to alter a column in a table right after you created it. 
Please just delete the NOT NULL constraint from the CREATE TABLE on line 125